src/lib/marshal/guide/types.ts

```diff
@@ -12,6 +13,11 @@ type GuideStepData = {
   fields: ContentSchemaField[];
```

Comment:
I missed this earlier, but I believe we should update this to be a "values" field.